src/mon/MgrStatMonitor.cc

```diff
@@ -79,7 +79,9 @@ void MgrStatMonitor::update_from_paxos(bool *need_bootstrap)
   version = get_last_committed();
   dout(10) << " " << version << dendl;
   bufferlist bl;
-  get_version(version, bl);
+  int err = get_version(version, bl);
```

Comment:
@tchaikov Could you please give more details about your comment ? I don't understand it (may be due to less knowledge in ceph project). I have discussed with Brad and he was also not able to clarify. Thanks.